### ISSUE TITLE
Intermittent error in `getdata` testset

### ISSUE THREAD

**Issue by koalive** [menchelab/BioProfiling.jl#25]
The following sometimes occur
```
getdata: Error During Test at /home/jovyan/test/runtests.jl:298
  Got exception outside of a @test
  ArgumentError: column name :Ft3 not found in the data frame; existing most similar names are: :Ft4
  Stacktrace:
   [1] lookupname at /opt/julia/packages/DataFrames/uPgZV/src/other/index.jl:253 [inlined]
   [2] getindex at /opt/julia/packages/DataFrames/uPgZV/src/other/index.jl:259 [inlined]
   [3] getindex(::DataFrame, ::typeof(!), ::Symbol) at /opt/julia/packages/DataFrames/uPgZV/src/dataframe/dataframe.jl:361
   [4] getproperty(::DataFrame, ::Symbol) at /opt/julia/packages/DataFrames/uPgZV/src/abstractdataframe/abstractdataframe.jl:295
   [5] top-level scope at /home/jovyan/test/runtests.jl:384
   [6] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1115
   [7] top-level scope at /home/jovyan/test/runtests.jl:303
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
```

**Comment by koalive** [comment]
Not observed with CI. This should be reopened if observed again.

**Comment by koalive** [comment]
Actually still an issue but should be corrected since 45495289d5d1a40eef8e6d96243335d19bd429d2
